corehq/apps/userreports/models.py

```diff
@@ -760,7 +766,40 @@ def ui_filters(self):
     @property
     @memoized
     def charts(self):
-        return [ChartFactory.from_spec(g._obj) for g in self.configured_charts]
+        configured_charts = deepcopy(self.configured_charts)
+        for chart in configured_charts:
+            if chart['type'] == 'multibar':
+                y_axis_columns = []
+                for y_axis_column in chart['y_axis_columns']:
+                    if isinstance(y_axis_column, dict):
+                        column_id = y_axis_column['column_id']
+                    else:
+                        column_id = y_axis_column
+                    column_config = self.report_columns_by_column_id[column_id]
+                    if column_config.type == 'expanded':
```

Comment:
using the same principle to expand on columns, as currently used when populating data rows

https://github.com/dimagi/commcare-hq/blob/c3468e680250c253c4bf17123a2d16ed0d53008d/corehq/apps/userreports/reports/util.py#L9-L18

https://github.com/dimagi/commcare-hq/blob/c3468e680250c253c4bf17123a2d16ed0d53008d/corehq/apps/userreports/reports/util.py#L81-L91